src/R/Editor/Impl/Selection/SelectWordCommand.cs

```diff
@@ -24,60 +22,13 @@ public sealed class SelectWordCommand : ViewCommand {
         public override CommandResult Invoke(Guid group, int id, object inputArg, ref object outputArg) {
```

Comment:
They do, but default one works for languages like C# and C++ (i.e. where dot is a separator).